client/src/glossary/glossary.js

```diff
@@ -146,7 +143,7 @@ const Glossary_ = ({ active_key, items_by_letter }) => (
                   <div className="glossary-top-link-container">
                     <a
                       className="glossary-top-link"
-                      href="#"
+                      href="#glossary"
                       tabIndex="0"
```

Comment:
This `tabIndex` isn't needed, I think it was there because an anchor with no href isn't focusable by default, probably from a time before this had the placeholder `href="#"`.